sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class Delay
+    {
+        protected Delay(System.TimeSpan? maxDelay = default(System.TimeSpan?), double jitterFactor = 0.2) { }
+        public static Azure.Core.Delay CreateExponentialDelay(System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.Delay CreateFixedDelay(System.TimeSpan? delay = default(System.TimeSpan?)) { throw null; }
+        public System.TimeSpan GetNextDelay(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        public System.Threading.Tasks.ValueTask<System.TimeSpan> GetNextDelayAsync(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        protected abstract System.TimeSpan GetNextDelayCore(Azure.Response? response, int retryNumber, System.Collections.Generic.IDictionary<string, object?> context);
```

Comment:
Should the nullable parameters take default values?  I feel like I'm more used to seeing nullable parameters come at the end of the signature.

Reply:
The reason that Response is nullable is that it is possible that there is no response received from the service. It is still a required parameter because GetNextDelay will always attempt to pass the response if it is there, and implementers expect the value to contain the response, if any.

Reply:
In terms of moving to the end of the signature, it felt more natural to have it be the first param as it is the primary data users are going to interact with. I suppose it could be swapped with retryNumber though - no strong preference here.